### ISSUE TITLE
Exec format error on ARM server

### ISSUE THREAD

**Issue by Xinayder** [caddyserver/ingress#132]
I have just tried to install the Ingress controller on my ARM server from Hetzner. Following the steps on the README, I got the following error right after the pod was deployed:

```bash
exec /ingress-controller: exec format error                                                                                      │
│ Stream closed EOF for caddy-system/mycaddy-caddy-ingress-controller-6c4b5b8f55-lqxgs (caddy-ingress-controller)
```

I forgot that I was running an ARM server so the next thing I checked was if the controller has an ARM image. To my surprise it has, but even after switching the image tag to `v0.1.5-arm64v8`, I get the same error.

The solution was to build my own image using: `sudo docker buildx build -t caddyingress-local .`

I thought it could be a build problem but I couldn't find anything. The only differences I have from the master branch is that I built the controller binary using `make` and had to change the last line of the Dockerfile to:

```Dockerfile
COPY bin/ingress-controller /
```

**Comment by Embraser01** [comment]
Hi 👋 

The docker image was badly built for ARM. The #128 PR should fix the issue if you'd like to give it a go (once I release a new version)

Fixed by #128

**Comment by mavimo** [comment]
I'm going to close the issue as it is resolved by the PR mentioned by @Embraser01.
